Demo/Demo/Networking/MerchantIntegration.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+enum MerchantIntegration {
```

Comment:
Will we add a UI option to toggle this setting in the future?

Reply:
Yes, we are still figuring out the UI in demo app that would help us to test different scenarios.